earn/src/pages/BoostPage.tsx

```diff
@@ -235,11 +265,17 @@ export default function BoostPage() {
 
   return (
     <AppPage>
-      <Text size='XL'>Boost</Text>
+      <ExplainerWrapper>
+        <Text size='M' weight='regular' color={SECONDARY_COLOR}>
+          Aloe Boost puts your Uniswap positions on steroids－earn up to 5x more swap fees and spend less on gas. Click
+          a lightning bolt to get started.
+        </Text>
+      </ExplainerWrapper>
+      <Text size='XL'>Positions</Text>
       <div className='flex flex-wrap gap-4 mt-4'>
         {isLoading &&
           allCardInfos.length === 0 &&
-          [...Array(4)].map((_, index) => <BoostCardPlaceholder key={index} />)}
+          [...Array(1)].map((_, index) => <BoostCardPlaceholder key={index} />)}
```

Comment:
Well ideally the number of placeholders would match the number of cards, but that's not possible. So I thought it looked better to go from too few --> too many than vice versa. If we go from 4 placeholders to 2 actual cards, for example, it might make the user think that something is based.